assets/app/widgets/crowbar-upgrade-nav/crowbar-upgrade-nav.directive.jade

```diff
@@ -1,7 +1,7 @@
 button.btn.btn-default.pull-left(type='button')
   span(aria-hidden="true")
   | Cancel Upgrade
-button.btn.btn-success.pull-right(type='button', ng-click="steps.nextStep()", ng-if="!steps.isLastStep()")
+button.btn.btn-success.pull-right(type='button', ng-click="steps.nextStep()", ng-if="!steps.isLastStep()", ng-class="{disabled : steps.activeStep.finished != true   }")
```

Comment:
Can you use `ng-disabled` instead?
Also, too many spaces before closing `ng-class` attribute